src/Stratis.Bitcoin/P2P/PeerAddressManager.cs

```diff
@@ -64,66 +61,83 @@ public PeerAddressManager(IDateTimeProvider dateTimeProvider, DataFolder peerFil
         public void LoadPeers()
         {
             List<PeerAddress> loadedPeers = this.fileStorage.LoadByFileName(PeerFileName);
+
             this.logger.LogTrace("{0} peers were loaded.", loadedPeers.Count);
 
-            loadedPeers.ForEach(peer =>
+            foreach (PeerAddress loadedPeer in loadedPeers)
             {
-                // Ensure that any address already in store is mapped.
-                peer.Endpoint = peer.Endpoint.MapToIpv6();
```

Comment:
Why did this get removed?

Reply:
Because now we do this check in AddPeer. Therefore all peers that are saved and loaded are already mapped to Ipv6

Reply:
Sweet 👍